walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,154 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeAllDay() {
```

Comment:
Good practice to have better test names:

https://docs.google.com/document/d/1WqjdTF5ifkBx-9Ls3T1V4reAdnh0DRe_WZcvYlmnBlk/edit?hl=en#heading=h.9vmuly4hjqyw